astropy/units/si.py

```diff
@@ -252,7 +252,7 @@
     ["deg_C", "Celsius"],
     namespace=_ns,
     doc="Degrees Celsius",
-    format={"latex": r"{}^{\circ}C", "unicode": "°C"},
+    format={"latex": r"{}^{\circ}C", "unicode": "°C", "fits": "Celsius"},
```

Comment:
Sometimes I am so pleasantly surprised at how the machinery just extends easily!